packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -577,6 +577,19 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
         this.autoscaledPoolOptions = { ...autoscaledPoolOptions, ...basicCrawlerAutoscaledPoolConfiguration };
     }
 
+    private getPeriodicLogger() {
+        const loggingInterval = 60e3;
+        const client = this.config.getStorageClient();
+
+        const log = async () => {
+            const currentStats = this.stats.calculate();
+            await client.setStatusMessage?.(`Crawled ${this.handledRequestsCount}/${currentStats.requestsTotal} pages.`);
```

Comment:
we wanted to have the messages a bit smarter, our initial thoughts were this:

- as long as the crawler runs normally, we could write something like `Crawling 5/123 pages, 8 requests failed, 5 results in default dataset.`, we’d have to define what means the “normally”, e.g. as long as there are no hard failures (requests that reached all retries), we could consider normal
- when we see higher error rate, we could print something about the errors, e.g. the most common error, maybe something from the retry histogram or statistics object?
- maybe some stats about average response time?

https://apifier.slack.com/archives/CG3EJESUX/p1671462495090689

Reply:
It's a bit updated / more granular now, but the final format is still WIP - imo in this case, less is more - status message should inform you about the run status as fast as possible - having long, convoluted messages filled with data isn't the way to go (again, just my opinion, I've seen the Slack thread)